### PR TITLE
Delete old examples/, rename linen_examples/ -> examples/

### PR DESCRIPTION
**Pull request by avital**
Keep the old LM1b example as examples/lm1b_deprecated, until #567
is resolved, per @levskaya's plan.

### CODE REVIEW

**Comment by avital** [comment]
This PR resolves #658

File: tests/run_all_tests.sh [hunk 1]
```diff
@@ -53,22 +53,27 @@ else
...
+    # TODO: Remove after github.com/google/flax/issues/567 is resolved
+    if [[ "$egd" == "examples/lm1b_deprecated" ]]; then
+        continue
+    fi
+
     # use cd to make sure pytpe cache lives in example dir and doesn't name clash
     # use *.py to avoid importing configs as a top-level import which leads tot import errors
     # because config files use relative imports (e.g. from config import ...). 
-    (cd $egd ; pytype "*.py")
+  if [[ "$OSTYPE" == "darwin"* ]]; then
```
Comment (andsteing) [inline tests/run_all_tests.sh:72]: Maybe extract this into a function if we're using it twice?
Currently it prints the same message for every example.

**Comment by andsteing** [comment]
(Also note there are a couple of conflicts that need to be resolved first)

File: tests/run_all_tests.sh (hunk 1 above)
Comment (avital) [reply tests/run_all_tests.sh:72]: I restructured this code to run pytype separately, and will file a new PR because this one became stale.

**Comment by avital** [comment]
Re-did this PR as https://github.com/google/flax/pull/884

**Pull request closed without merging**